quickstart/openvino-serving-runtime.yaml

```diff
@@ -15,8 +15,8 @@ apiVersion: serving.kserve.io/v1alpha1
 kind: ServingRuntime
 metadata:
   annotations:
-    create-route: "True"
-    enable-auth: "True"
+    enable-route: "true"
```

Comment:
should these also be changed? https://github.com/opendatahub-io/modelmesh-serving/blob/main/tests/resources/modelmesh/serving-runtime.yaml#L18-L19